lib/corr.rb

```diff
@@ -1,44 +1,42 @@
 require "corr/version"
 
 module Corr
-  class Formulas
```

Comment:
Why do you delete the class inside the module??

Reply:
The gem doesn't need it. Works fine without the class. Plus short invokes.